deployment/zerops.md

```diff
@@ -1,7 +1,159 @@
 ---
 title: Deploying on Zerops
-subtitle: Run Gleam with Ease. No ops required. 
+subtitle: Run Gleam with Ease. Zero Ops required. 
 layout: page
 redirect_from:
   - "/deploying-gleam-on-zerops/index.html"
 ---
+
+
+[Zerops](https://zerops.io) is a dev-first cloud platform and easy to use and economically viable deployment platform with [free $15 credits](https://zerops.io/#pricing) no credit card required. You can get $50 more after adding $10 into your zerops account.
+
+Zerops' free allowance of $15 is enough to deploy any type of full stack project without any credit card.
+
+## Prepare an application
+
+We will need a Gleam backend web application to deploy in this guide, so first
+we will make one. If you already have an application skip to the next section.
+
+Create a new Gleam project and add the required dependencies.
+
+```sh
+gleam new my_web_app
+cd my_web_app
+gleam add mist gleam_http gleam_erlang
+```
+
+Open up `src/my_web_app.gleam` and replace the contents with this code that
+defines a micro web application.
+
+```gleam
+import mist
+import gleam/erlang/process
+import gleam/bytes_builder
+import gleam/http/response.{Response}
+
+pub fn main() {
+  let assert Ok(_) =
+    web_service
+    |> mist.new
+    |> mist.port(8080)
+    |> mist.start_http
+  process.sleep_forever()
+}
+
+fn web_service(_request) {
+  let body = bytes_builder.from_string("Hello, Zerops!")
+  Response(200, [], mist.Bytes(body))
+}
+```
+
+Now we have a web application that listens on port 8080 and can be started with
+`gleam run`.
+
+## Gleam x Zerops Quickrun
+Want to test Gleam on Zerops without installing or setting up anything? Use our gleam recipe [Zerops x Gleam](https://github.com/zeropsio/recipe-gleam) using the project import yaml mentioned below or Deploy with a Single Click.
+
+```sh
+project:
+  name: recipe-gleam
+services:
+  - hostname: app
+    type: rust@1
+    enableSubdomainAccess: true
+    buildFromGit: https://github.com/zeropsio/recipe-nuxt-static
+```
+
+
+## Project Creation
+Projects and services can be added either through a [Project add](https://app.zerops.io/dashboard/project-add) wizard or imported using a yaml structure:
+
+```sh
+# see https://docs.zerops.io/references/import for full reference
+project:
+  name: recipe-gleam
+services:
+  - hostname: app
+    type: rust@1
+    enableSubdomainAccess: true
+```
+
+This will create a project called `recipe-gleam` with a Zerops Static service called `app`.
+
+
+## Add Zerops.yml
+
+```yml
+zerops:
+  - setup: app 
+    build:
+      base: rust@1
```

Comment:
How come Rust is used as a base? Gleam apps don't depend on Rust

Reply:
Yes you're right, i'll fix everything ☺️ Thanks for pointing it out 💯

Reply:
This is embarrassing NGL 😨

Reply:
Rust is used as a base because they offer pre-prepared language images and Ubuntu. The images default to Alpine, I assume Alpine + Rust + Gleam is slimmer than Ubuntu + Gleam. I'm not sure if Rust is the absolute smallest image offered.